src/loaders/utils/__tests__/resolveES6.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { generate } from 'escodegen';
+import { builders as b } from 'ast-types';
+import resolveESModule from '../resolveESModule';
+
+it('resolveES6 should return an array of AST', () => {
```

Comment:
```suggestion
it('should return an array of AST', () => {
```

And we need to rename the file too.